app/views/admin/users/index.haml

```diff
@@ -64,6 +64,23 @@
                   %div= "Sign-in provided by #{user.provider.upcase}"
                 = render 'user_invitation_status', user: user
               %td
+                - if user.invitation_status != :active || user.access_locked?
+                  .mb-6
+                    - if user.access_locked?
+                      .w-100
+                        = link_to unlock_admin_user_path(user), method: :post, class: 'btn btn-sm btn-secondary mb-2' do
+                          %i.icon-unlocked
+                          Unlock
+                    - if user.invitation_status == :pending_confirmation
+                      .w-100
+                        = link_to admin_user_resend_invitation_path(user), method: :post, class: 'btn btn-sm btn-secondary mb-2' do
+                          %i.icon-envelope-o
+                          Resend Invitation
+                    - if user.invitation_status == :invitation_expired
```

Comment:
Ah your comment made me realize that I changed the behavior slightly here. Previously we only showed the "Unlock" button if the user had an active invitation status. Here, it would show the "Unlock" button even if the users invitation is pending or expired. I'll change it back to the previous mutually exclusive behavior:
```
if user.invitation_status == :pending_confirmation
   pass
elsif user.invitation_status == :invitation_expired
   pass
elsif user.access_locked?
   pass
```